concent_api/core/tests/test_integration_force_payment.py

```diff
@@ -181,6 +183,69 @@ def test_provider_send_force_payment_with_subtask_results_accepted_where_ethereu
         )
         self._assert_stored_message_counter_not_increased()
 
+    def test_that_if_provider_sends_force_payment_with_two_different_providers_ethereum_accounts_concent_should_refuse(self):
+        """
+        Expected message exchange:
+        Provider  -> Concent:    ForcePayment
+        Concent   -> Provider:   ServiceRefused
+        """
+        report_computed_task1 = self._get_deserialized_report_computed_task(
+            timestamp="2018-02-05 10:00:05",
+            task_to_compute=self._get_deserialized_task_to_compute(
+                timestamp="2018-02-05 10:00:00",
+                deadline="2018-02-05 10:00:10",
+                subtask_id=self._get_uuid('1'),
+                price=15000,
+            ))
+
+        provider1_keys = cryptography.ECCx(None)
+        provider_ethereum_public_key = encode_hex(provider1_keys.raw_pubkey)
```

Comment:
Why not using `self._get_provider_ethereum_hex_public_key_different()` ?

Reply:
I can do it as well, done :)